src/main/resources/application.yml

```diff
@@ -57,3 +57,9 @@ management:
       exposure:
         include: info, health
       base-path: /public/actuator
+
+ays:
+  scheduler:
+    UnusedInvalidTokenDeletionScheduler:
+      cron: '0 0 */3 * * ?'
+      enabled: true
```

Comment:
Parametreleri burada yönetmek zorunda değiliz, değişken olabilecek bir yapı değil

Reply:
Test ortamında kısaltıp test edebilmek için parametre olarak eklendi.